awscli/customizations/commands.py

```diff
@@ -136,6 +143,13 @@ def __call__(self, args, parsed_globals):
             ".".join(self.lineage_names)
         self._session.emit(event, argument_table=self._arg_table, args=args,
                            session=self._session)
+        maybe_parsed_subcommand = self._parse_potential_subcommand(
+            args, self._subcommand_table
+        )
+        if maybe_parsed_subcommand is not None:
```

Comment:
With this if statement now added, does this line ever get used? https://github.com/aws/aws-cli/pull/7399/files#diff-ae84646454f40103f091a890d7a1f826eaebc333454d1a5772ce31c721881cd6R211 It seems like it would not (e.g. in the `s3` subcommand).